R/verify.R

```diff
@@ -0,0 +1,85 @@
+#' Check Code Reproducibility
+#'
+#' Checks whether code in `teal_data` reproduces the stored objects.
+#'
+#' @return If the objects returned by the `@code` slot in the input `teal_data` object exactly match
+#' the objects in `teal_data@env`, the function updates the `@verified` slot to `TRUE` in the `teal_data` object.
+#' Once it's verified, the slot will always be set to `TRUE`. If the `@code` fails to recreate objects in
+#' `teal_data@env`, an error is thrown.
```

Comment:
Some of this information belongs in `Details`.

Reply:
Yeah, this was condensed details section included in Value/return section. I changed it here 7b965199faa455436e8d8f012e5cef31240d8e37 so all the details are moved to details, and Value/return only corresponds to the class of the object.